socorro/unittest/external/es/base.py

```diff
@@ -138,55 +136,46 @@ def get_base_config(self, cls=ConnectionContext, es_index=None):
 
         return self.get_tuned_config(cls, extra_values=extra_values)
 
-    def index_crash(self, processed_crash=None, raw_crash=None, crash_id=None):
+    def index_crash(
+        self, processed_crash=None, raw_crash=None, crash_id=None, refresh=True
+    ):
+        """Index a single crash and refresh"""
```

Comment:
This used to access Elasticsearch directly which means that indexing crashes didn't go through our crash indexing code and thus the tests were pretty seriously diverged from reality.

I reworked this and `index_crash_bulk` to use the crashstorage code to save crashes.